srcpkgs/ipv6calc/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'ipv6calc'
+pkgname=ipv6calc
+version=2.0.0
+revision=1
+build_style=gnu-configure
+configure_args="--enable-geoip"
+hostmakedepends="perl"
+makedepends="geoip-devel"
+depends="geoip"
```

Comment:
should be found via shlibs

Reply:
or rather libgeoip

Reply:
Right you are. Changed.

Reply:
i'll assume you checked that it doesn't use any of the executables provided by `geoip` and `libgeoip` is enough.

Reply:
Yes. I wasn't really thinking when I added geoip to the dependencies. It should have been libgeoip.